gpytorch/lazy/cached_cg_lazy_tensor.py

```diff
@@ -97,12 +97,12 @@ def __init__(
             probe_vector_tmats=probe_vector_tmats,
         )
         self.base_lazy_tensor = base_lazy_tensor
-        self.eager_rhss = [eager_rhs.requires_grad_(False) for eager_rhs in eager_rhss]
-        self.solves = [solve.requires_grad_(False) for solve in solves]
-        self.probe_vectors = probe_vectors.requires_grad_(False)
-        self.probe_vector_norms = probe_vector_norms.requires_grad_(False)
-        self.probe_vector_solves = probe_vector_solves.requires_grad_(False)
-        self.probe_vector_tmats = probe_vector_tmats.requires_grad_(False)
+        self.eager_rhss = [eager_rhs.detach_().requires_grad_(False) for eager_rhs in eager_rhss]
```

Comment:
`detach_().requires_grad_(False)` -> `detach()`, also below

Reply:
I went ahead and fixed this because I need this updated for a small PR I am working on anyways that makes `InvMatmul` twice differentiable.